soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,25 @@
+import single_robot_sequence
+import role_assignment
+import skills.move
+import behavior
+
+#Moves a SINGLE robot in a sequence of points
+class MoveSequence(single_robot_sequence.SingleRobotSequence):
```

Comment:
I was mostly using the MoveSequence as an example of how to implement a behavior sequence, but that is a clever way of doing it. Are there any cases that would make a good example of when to use behavior sequence?